seed/campaigns.ts

```diff
@@ -57,3 +61,34 @@ export default async function seedCampaigns(
 
   return campaignIds
 }
```

Comment:
Let's see if we can't get this down to a single for-loop, or `.map()` expression calling a _single_ method.

Reply:
Oh.. I spoke w/ @oakinh on Slack and presented it to him as a "if you want a challenge..." kind of thing...